theano/sandbox/cuda/basic_ops.py

```diff
@@ -3329,20 +3329,35 @@ def c_code_cache_version(self):
         return ()
 
 
-class GpuFlatten(gof.HideC, tensor.Flatten, GpuOp):
+class GpuFlatten(gof.HideC, tensor.Reshape, GpuOp):
     """
     Implement Flatten on the gpu.
 
```

Comment:
There should be a note in the docstring. Like [here](https://github.com/Theano/Theano/blob/master/theano/tensor/basic.py#L4047).